### ISSUE TITLE
Unable to upload most PDFs

### ISSUE THREAD

**Issue by adamklaff** [ciur/papermerge#309]
**Description**
I am able to upload some PDFs, but most times when I upload, the upload never completes and I get an error. This occurs when I drag and drop onto any upload window or use the "upload" button. Sometimes I get an "error-missing input file" error in the upload modal. 

**Expected**
I expect the upload to complete.

**Actual**
Upload doesn't complete. I get the following errors in the JS console:
```
Uncaught SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at XMLHttpRequest.<anonymous> (papermerge.js:54)
(anonymous) @ papermerge.js:54

Failed to load resource: net::ERR_CONNECTION_ABORTED

```

**Info:**
 - OS: Windows 10 and Mac OS Catalina
 - Chrome
-  Database Postgres 13.2
 - Papermerge Version 1.5.5

**Comment by ciur** [comment]
@adamklaff, Did you use docker image provided by linuxserver.io?

**Comment by adamklaff** [comment]
Yes. Should I post there? I realize you ask that up front but I wasn't sure
how that image could cause this problem. Thanks.

**Comment by jimbogr77** [comment]
I have the same issue. Is there any update? 
I'm using docker (official image) and Edge Chromium.

**Comment by ciur** [comment]
I cannot reproduce :(
Does issue occur for all PDFs ? For specific ones ? For Windows OS only? Is it same for Firefox browser ? What about jpeg, png files - do they upload successfully ?

**Comment by jimbogr77** [comment]
> I cannot reproduce :(
> Please help me with more information about the issue.
> Does issue occur for all PDFs ? For specific ones ? For Windows OS only? Is it same for Firefox browser ? What about jpeg, png files - do they upload successfully ?

Hi, apologies for the late reply, I have missed the notification. 

These are 2 PDFs that won't upload:

https://1drv.ms/u/s!AvUWNyiOClNSiN9h9c6uz_V3yN8fsA?e=hzoeQ3

Network console logs from firefox:
[network console logs.txt](https://github.com/ciur/papermerge/files/6324147/network.console.logs.txt)



They are both scanned, but at the same time other scanned documents are uploading just fine.

Same issue on Edge Chromium and Firefox (latest versions)

**Comment by barbados-clemens** [comment]
I just came across this myself and it was because it was a password protected PDF. It doesn't seem to be the same as others are seeing. But providing an api response for 500 error and surface in the UI might be helpful.

**Comment by Dzheky** [comment]
Same issue, getting 500 for some PDFs, but some uploads fine, can't find the pattern though

**Comment by Dominik-Koenig** [comment]
I started using papermerge today and I had the same issue. 
But this error does not seem to be bound to the pdf file itself, for me it happened when I dropped multiple files to papermerge, some uploaded, some didn't. However, if I dropped the files again as a single file the issue did not appear. Maybe it's an issue with the file sizes or request size?
